content/en/providers/joining/federated-resource-centre/_index.md

```diff
@@ -181,8 +181,7 @@ International communities reach EGI through the following channels:
 - The EGI site where they can
   [request access to services](https://www.egi.eu/services/research/).
 - The [EGI-ACE Call for Use Cases](https://www.egi.eu/egi-ace-open-call/).
-- The [Marketplace](https://marketplace.eosc-portal.eu/) of the European Open
-  Science Cloud.
+- The [European Open Science Cloud](https://open-science-cloud.ec.europa.eu/).
```

Comment:
Is EGI easily reachable through this web? I would remove this one

Reply:
Agreed and done. Thanks!